### ISSUE TITLE
iOS UIWebview scroll, promise polyfill bug

### ISSUE THREAD

**Issue by ustccjw** [zloirock/core-js#339]
relative issuse: 

* https://github.com/petkaantonov/bluebird/issues/666
* https://github.com/vuejs/vue/issues/6054

**Comment by ustccjw** [comment]
I like `PromiseRejectionEvent ` polyfill, but iOS do not implement `PromiseRejectionEvent `. But it seems promise polyfill has bug in iOS UIWebview when scrolling.

**Comment by ustccjw** [comment]
iOS 11 has support PromiseRejectionEvent

**Comment by zloirock** [comment]
Fixed in `core-js@2.5.2`.

**Comment by ustccjw** [comment]
```js
Navigator.standalone 
Returns a boolean indicating whether the browser is running in standalone mode. Available on Apple's iOS Safari only.
```
this problem is happened in iOS UIWebview, not only iOS Safari
